apps/web/middleware.ts

```diff
@@ -41,6 +41,24 @@ const middleware = async (req: NextRequest): Promise<NextResponse<unknown>> => {
     }
   }
 
+  if (url.pathname.startsWith("/apps/routing-forms")) {
+    let redirectPathName = "";
+
+    if (url.pathname.startsWith("/apps/routing-forms/forms")) {
+      redirectPathName = url.pathname.replace("/apps/routing-forms/forms", "/routing");
+    } else {
+      redirectPathName = url.pathname.replace("/apps/routing-forms", "/routing");
+    }
+    url.pathname = redirectPathName;
+    return NextResponse.redirect(url, { headers: requestHeaders });
+  }
+
+  if (url.pathname.startsWith("/routing/forms")) {
+    const redirectPathName = url.pathname.replace("/routing/forms", "/routing");
+    url.pathname = redirectPathName;
+    return NextResponse.redirect(url, { headers: requestHeaders });
+  }
+
```

Comment:
Why are we making it overcomplicated by using so many conditions while combining the first two conditions into a single block makes it much more readable?